### PR TITLE
Add notification sound when paratroopers land

### PR DESCRIPTION
**Pull request by gkaretka**
https://github.com/OpenRA/OpenRA/issues/14003

### CODE REVIEW

**Comment by GraionDilach** [comment]
This probably needs an IsNullOrEmpty check.

**Comment by gkaretka** [comment]
It was caused by whitespace on the end of line

File: OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs [hunk 1]
```diff
@@ -43,6 +43,7 @@ public abstract class SupportPowerInfo : PausableConditionalTraitInfo
 		public readonly string LaunchSpeechNotification = null;
 		public readonly string IncomingSound = null;
 		public readonly string IncomingSpeechNotification = null;
+		public readonly string ReinforcementsArrivedSpeechNotification = null;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: Why did you add this to the general `SupportPowerInfo` and not to `ParatroopersPowerInfo`? (As only the `ParatroopersPower` uses it anyway.)

Comment (gkaretka) [reply OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: In `ParatroopersPowerInfo` there are no sound variables so I thought it would be better to have it in one place

Comment (GraionDilach) [reply OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: It's just clutter for anything else but ParatroopersPower though so I agree it would be better in PPInfo.

Comment (gkaretka) [reply OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: Ok, I'll fix it

File: OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs [hunk 2]
```diff
@@ -113,6 +115,8 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 				}
 
 				aircraftInRange[a] = true;
+				Game.Sound.PlayNotification(self.World.Map.Rules, self.Owner, "Speech",
```
Comment (abc013) [inline OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs]: The `IsNullOrEmpty` check is gone.

Comment (gkaretka) [reply OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs]: It's not needed there because of check in function

		public bool PlayNotification(Ruleset rules, Player player, string type, string notification, string variant)
		{
			if (rules == null)
				throw new ArgumentNullException("rules");

			if (type == null || notification == null)
				return false;

			return PlayPredefined(SoundType.UI, rules, player, null, type.ToLowerInvariant(), notification, variant, true, WPos.Zero, 1f, false);
		}

Comment (abc013) [reply OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs]: Oh, ok. Thanks.

File: OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs [hunk 3]
```diff
@@ -26,6 +26,8 @@ public class ParatroopersPowerInfo : SupportPowerInfo
 		public readonly int SquadSize = 1;
 		public readonly WVec SquadOffset = new WVec(-1536, 1536, 0);
 
+		public readonly string ReinforcementsArrivedSpeechNotification = null;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs]: What about adding a `[Desc("Notification to play when entering the drop zone.")]`?

**Review by abcdefg30 — COMMENTED** [review]
Hm, I think this currently plays the notification each time a unit enters the drop zone to paradrop units, which means three notifications when you use three badgers to drop units. (Didn't test that though.) We should think about that. (Maybe limit the notification to one notification per use of the power?)

**Review by pchote — APPROVED** [review]
Code changes LGTM, but can you please squash down to one commit?

**Review by pchote — CHANGES_REQUESTED** [review]
> Hm, I think this currently plays the notification each time a unit enters the drop zone to paradrop units, which means three notifications when you use three badgers to drop units

Issue confirmed.

This can be fixed by moving it above the `aircraftInRange[a] = true` and adding a `!aircraftInRange.Any(kv => kv.Value)` check.

**Comment by gkaretka** [comment]
@pchote hi, I pulled bleed and it caused some error which is not causing my PR
        Testing Tiberian Sun mod MiniYAML...

        Testing mod: Tiberian Sun

        OpenRA.Utility(1,1): Error: Failed with exception: System.IO.FileNotFoundException: File not found: ^maps/ts/{DEV_VERSION}

        File name: '^maps/ts/{DEV_VERSION}'

          at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00031] in /home/travis/build/OpenRA/OpenRA/OpenRA.Game/FileSystem/FileSystem.cs:216 

          at OpenRA.FileSystem.FileSystem.OpenPackage (System.String filename) [0x00051] in /home/travis/build/OpenRA/OpenRA/OpenRA.Game/FileSystem/FileSystem.cs:74 

          at OpenRA.Mods.Common.UtilityCommands.CheckYaml.OpenRA.IUtilityCommand.Run         (OpenRA.Utility utility, System.String[] args) [0x001f3] in /home/travis/build/OpenRA/OpenRA        /OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs:93 

        make: *** [test] Error 1

**Comment by pchote** [comment]
That will be fixed by #14136.  For now, just keep this branch based before the #13938 merge commit.
